host_core/lib/host_core/actors/actor_module.ex

```diff
@@ -192,24 +205,23 @@ defmodule HostCore.Actors.ActorModule do
 
       # TODO - in the future, poll these so we can forward the err/out pipes
       # to our logger
-      {:ok, stdin} = Wasmex.Pipe.create()
-      {:ok, stdout} = Wasmex.Pipe.create()
-      {:ok, stderr} = Wasmex.Pipe.create()
+      {:ok, stdin} = Wasmex.Pipe.new()
+      {:ok, stdout} = Wasmex.Pipe.new()
+      {:ok, stderr} = Wasmex.Pipe.new()
```

Comment:
These look duplicated now? Aren't we already creating these pipes on line 186?

Reply:
Doof, thank you. I was going too fast to get this up before the community call >.<